include/serial/impl/win.h

```diff
@@ -198,6 +198,9 @@ class serial::Serial::SerialImpl {
   HANDLE read_mutex;
   // Mutex used to lock the write functions
   HANDLE write_mutex;
+
+  OVERLAPPED ov_read;           // OVERLAPPED read
```

Comment:
ov_read and ov_write are not fully intialized, some unused members have random values. I need to zero those to avoid ERROR_INVALID_PARAMETER at runtime:
```
  OVERLAPPED ov_read = { 0 };           // OVERLAPPED read
  OVERLAPPED ov_write = { 0 };          // OVERLAPPED write
```
MSDN: "Any unused members of this structure should always be initialized to zero before the structure is used in a function call. Otherwise, the function may fail and return ERROR_INVALID_PARAMETER."
(https://learn.microsoft.com/en-us/windows/win32/api/minwinbase/ns-minwinbase-overlapped?devlangs=cpp&f1url=%3FappId%3DDev16IDEF1%26l%3DEN-US%26k%3Dk(MINWINBASE%252FOVERLAPPED)%3Bk(OVERLAPPED)%3Bk(DevLang-C%252B%252B)%3Bk(TargetOS-Windows)%26rd%3Dtrue)